### DISCUSSION TITLE
Trying to get it to work but need some help

### DISCUSSION THREAD

**Discussion by nitrobe** [berarma/new-lg4ff#71] [Q&A]
sudo dkms install /usr/src/new-lg4ff
Creating symlink /var/lib/dkms/new-lg4ff/0.4.1/source -> /usr/src/new-lg4ff-0.4.1

Kernel preparation unnecessary for this kernel. Skipping...

Building module:
cleaning build area...(bad exit status: 2)
make -j16 KERNELRELEASE=5.15.0-56-generic KVERSION=5.15.0-56-generic...(bad exit status: 2)
ERROR (dkms apport): binary package for new-lg4ff: 0.4.1 not found
Error! Bad return status for module build on kernel: 5.15.0-56-generic (x86_64)
Consult /var/lib/dkms/new-lg4ff/0.4.1/build/make.log for more information.

cat /var/lib/dkms/new-lg4ff/0.4.1/build/make.log
DKMS make.log for new-lg4ff-0.4.1 for kernel 5.15.0-56-generic (x86_64)
Fr 30 Dez 2022 13:56:42 CET
make: *** No targets specified and no makefile found.  Stop.

**Comment by berarma** [comment]
Does the file `/usr/src/new-lg4ff/Makefile` exist and is readable?

Which distro are you using?

**Reply by nitrobe** [reply]
ubuntu 22

i put  the makefile in that dir

Creating symlink /var/lib/dkms/new-lg4ff/0.4.1/source -> /usr/src/new-lg4ff-0.4.1

Kernel preparation unnecessary for this kernel. Skipping...

Building module:
cleaning build area...
make -j16 KERNELRELEASE=5.15.0-56-generic KVERSION=5.15.0-56-generic...
ERROR (dkms apport): binary package for new-lg4ff: 0.4.1 not found
Error!  Build of hid-logitech-new.ko failed for: 5.15.0-56-generic (x86_64)
Make sure the name of the generated module is correct and at the root of the
build directory, or consult make.log in the build directory
/var/lib/dkms/new-lg4ff/0.4.1/build/ for more information.

cat /var/lib/dkms/new-lg4ff/0.4.1/build/make.log
DKMS make.log for new-lg4ff-0.4.1 for kernel 5.15.0-56-generic (x86_64)
Fr 30 Dez 2022 18:27:25 CET
make -C /lib/modules/5.15.0-56-generic/build M=$PWD
make[1]: Entering directory '/usr/src/linux-headers-5.15.0-56-generic'
  MODPOST /var/lib/dkms/new-lg4ff/0.4.1/build/Module.symvers
make[1]: Leaving directory '/usr/src/linux-headers-5.15.0-56-generic'

**Reply by berarma** [reply]
You don't have to put the makefile in that directory, you have to put the full contents of this repository.

Please, follow the instructions in the Readme.

**Reply by nitrobe** [reply]
sudo mv *.* did not move every file.

tried it again 

sudo dkms install /usr/src/new-lg4ff
Creating symlink /var/lib/dkms/new-lg4ff/0.4.1/source -> /usr/src/new-lg4ff-0.4.1

Kernel preparation unnecessary for this kernel. Skipping...

Building module:
cleaning build area...
make -j16 KERNELRELEASE=5.15.0-56-generic KVERSION=5.15.0-56-generic...
Signing module:
 - /var/lib/dkms/new-lg4ff/0.4.1/5.15.0-56-generic/x86_64/module/hid-logitech.ko
Secure Boot not enabled on this system.
cleaning build area...

hid-logitech.ko:
Running module version sanity check.
 - Original module
 - Installation
   - Installing to /lib/modules/5.15.0-56-generic/updates/dkms/

depmod...

update-initramfs: Generating /boot/initrd.img-5.15.0-56-generic

rebooted and still not showing in oversteer

**Reply by nitrobe** [reply]
couple of reboots later and its working!
